### ISSUE TITLE
The upgrade to phpdotenv 3 seems to have broken compatibility with Laravel 5.5

### ISSUE THREAD

**Issue by JeroenVanOort** [Codeception/Codeception#5389]
#### What are you trying to achieve?
I'm trying to upgrade codeception to version 2.5.3 in a Laravel 5.5 project and run codeception afterwards.

#### What do you get instead?
When running codecept I get a fatal error.
> Provide console output if related. Use `-vvv` mode for more details.

```bash
vendor/bin/codecept -vvv

Fatal error: Uncaught Error: Call to undefined method Dotenv\Dotenv::create() in /var/www/vendor/codeception/codeception/src/Codeception/Lib/ParamsLoader.php:118
```

### Details

* Codeception version: 2.5.3
* PHP Version: 7.2.10
* Operating System: Ubuntu 18.04
* Installation type: Composer

As Laravel 5.5 has a `~2.2` version specification for it's phpdotenv dependency, it will install the latest 2 version, being 2.6.1. And since codeception 2.5.3 has a `^3.0` dev-dependency, that won't collide with Laravel's dependency. But codeception does depend on functions that are not available with phpdotenv 2, even in non-development.

This is what I've found, but please let me know if there's another cause for this problem.

**Comment by edno** [comment]
I confirm the same. 

The conflict occurs with the package [lavarel/framework](https://github.com/laravel/framework): https://github.com/laravel/framework/blob/5.7/composer.json#L43

**Comment by edno** [comment]
It will be fixed by #5380

**Comment by edno** [comment]
Resolved in 2.5.4
